includes/installer.php

```diff
@@ -36,15 +36,15 @@ public function install() {
         if ( false === get_option( 'wpbdp-db-migrations', false ) )
             update_option( 'wpbdp-db-migrations', array(), false );
 
-        if ( self::DB_VERSION == $this->installed_version )
+		if ( version_compare( self::DB_VERSION, $this->installed_version, '=' ) )
```

Comment:
@stephywells please let me know if this is okay to compare the DB versions or if I should revert

Reply:
That looks fine to me. Thanks!